### DISCUSSION TITLE
Spawner Mode

### DISCUSSION THREAD

**Discussion by andi-makes** [SchmarrnDevs/Lighty#8] [Ideas and Suggestions]
Spawners require different light levels compared to normal spawns.

How would one enable Spawner mode though? How would it look like?

Ideas are welcome!

**Comment by mhmatthewhugley** [comment]
When I looked at your guys list of colors,
```
    Green: No mobs can spawn here!
    Orange: If the sun goes down, Mobs will spawn! Be careful!
    Red: Mobs can spawn here!
```
I noticed there is no color indicating where mobs can spawn from a spawner.
Would be a very helpful indicator to have.

Only other mod I know of that does this is https://www.curseforge.com/minecraft/mc-mods/light-overlay
